coulttxml

```diff
@@ -41,6 +41,27 @@ def count_molecules_from_pdb(pdb_filename):
         numbers[i] = numbers[i] - numbers[i-1]
     return numbers
 
+def count_molecules_from_pdbx(pdbx_filename):
```

Comment:
In the PDBx we should presume of the order or content of the columns, so it is more robust to parse the input file to find which field contains the molecule number, and give an error if not there. This works as is because the config comes from fftool and we know what we put there, but is not the best practice.